spec/models/user_spec.rb

```diff
@@ -916,7 +916,7 @@
 
           wf_double = double
           expect(wf_double).to receive(:fetch)
-          expect(Webfinger).to receive(:new).with('one').and_return(wf_double)
+          expect(Diaspora::Webfinger).to receive(:new).with('one').and_return(wf_double)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.